### ISSUE TITLE
Create a Test Suite for TEST TRAPI validation

### ISSUE THREAD

**Issue by maximusunc** [NCATSTranslator/Tests#23]

**Comment by RichardBruskiewich** [comment]
@maximusunc, do you mean a "reasoner-validator" kind of TestRunner? That testing framework validations TRAPI I/O and also, Biolink Model compliance...

**Comment by maximusunc** [comment]
Yes, I believe so. According to the Test Run Schedule: https://docs.google.com/document/d/1UNX7Z4Wjwg0FPA58VBNMYducNCq44LykzQ6_JU7FEEo/edit

**Comment by RichardBruskiewich** [comment]
Roger that! I'll put that at the front of my things to do list tomorrow morning (2024 March 6th)
